<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="zh-cn">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">691f9b9e1fcf24dd544d6fcae768f4dd0d552a7f</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Security.XmlSyntaxException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">633c1e5f4b0a7c992599c1ba5a664bd9e476e59e</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">2011bdc7a034477702ccc3789accf6aadc220b30</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when there is a syntax error in XML parsing.</source>
          <target state="translated">XML 分析中存在语法错误时引发的异常。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve">
          <source>This class cannot be inherited.</source>
          <target state="translated">此类不能被继承。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>XmlSyntaxException uses the default &lt;xref:System.Object.Equals%2A&gt; implementation, which supports reference equality.</source>
          <target state="translated">XmlSyntaxException 使用默认&lt;xref:System.Object.Equals%2A&gt;支持引用相等性的实现。&lt;/xref:System.Object.Equals%2A&gt;</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>For a list of initial property values for an instance of XmlSyntaxException, see the constructor.</source>
          <target state="translated">XmlSyntaxException 的实例的初始属性值的列表，请参阅构造函数。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with default properties.</source>
          <target state="translated">初始化的新实例<bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>使用默认属性的类。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Security.XmlSyntaxException&gt;.</source>
          <target state="translated">下表显示&lt;xref:System.Security.XmlSyntaxException&gt;.&lt;/xref:System.Security.XmlSyntaxException&gt;实例的初始属性的值</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|"Invalid syntax."|</source>
          <target state="translated">|属性 |值 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|  |&lt;xref:System.Exception.Message%2A&gt;|"语法无效。"|&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the line number where the exception was detected.</source>
          <target state="translated">初始化的新实例<bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>类异常检测到的行号。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>The line number of an XML stream is updated whenever a newline character is detected.</source>
          <target state="translated">当检测到换行字符时，将更新的 XML 流的行号。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>Numbering begins with 1 for the first line.</source>
          <target state="translated">从 1 开始编号的第一行。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Security.XmlSyntaxException&gt;.</source>
          <target state="translated">下表显示&lt;xref:System.Security.XmlSyntaxException&gt;.&lt;/xref:System.Security.XmlSyntaxException&gt;实例的初始属性的值</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|"Invalid syntax on line <ph id="ph2">`lineNumber`</ph>."|</source>
          <target state="translated">|属性 |值 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|  |&lt;xref:System.Exception.Message%2A&gt;|"在行上的无效语法<ph id="ph2">`lineNumber`</ph>。"|&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>The line number of the XML stream where the XML syntax error was detected.</source>
          <target state="translated">XML 流中检测到的 XML 语法错误的行号。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message.</source>
          <target state="translated">初始化的新实例<bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>与指定的错误消息的类。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Security.XmlSyntaxException&gt;.</source>
          <target state="translated">下表显示&lt;xref:System.Security.XmlSyntaxException&gt;.&lt;/xref:System.Security.XmlSyntaxException&gt;实例的初始属性的值</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">|属性 |值 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|  |&lt;xref:System.Exception.Message%2A&gt;|错误消息字符串。 |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">解释异常原因的错误消息。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and the line number where the exception was detected.</source>
          <target state="translated">初始化的新实例<bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>使用指定的错误消息和异常检测到的行号的类。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve">
          <source>The line number of an XML stream is updated whenever a newline character is detected.</source>
          <target state="translated">当检测到换行字符时，将更新的 XML 流的行号。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>Numbering begins with 1 for the first line.</source>
          <target state="translated">从 1 开始编号的第一行。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Security.XmlSyntaxException&gt;.</source>
          <target state="translated">下表显示&lt;xref:System.Security.XmlSyntaxException&gt;.&lt;/xref:System.Security.XmlSyntaxException&gt;实例的初始属性的值</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|"Invalid syntax on line <ph id="ph2">`lineNumber`</ph><ph id="ph3"> - </ph><ph id="ph4">`message`</ph> "|</source>
          <target state="translated">|属性 |值 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|  |&lt;xref:System.Exception.Message%2A&gt;|"在行上的无效语法<ph id="ph2">`lineNumber`</ph> <ph id="ph3"> - </ph> <ph id="ph4">`message`</ph> "|&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve">
          <source>The line number of the XML stream where the XML syntax error was detected.</source>
          <target state="translated">XML 流中检测到的 XML 语法错误的行号。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">解释异常原因的错误消息。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">初始化的新实例<bpt id="p1">&lt;xref href="System.Security.XmlSyntaxException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>使用指定的错误消息和对导致此异常的内部异常的引用的类。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">前一个异常的直接结果应包含中的前一个异常的引用引发的异常，&lt;xref:System.Exception.InnerException%2A&gt;属性。&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">&lt;xref:System.Exception.InnerException%2A&gt;属性返回相同的值传递到构造函数中，或<ph id="ph1">`null`</ph>如果&lt;xref:System.Exception.InnerException%2A&gt;属性不会提供给构造函数的内部异常值。&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Security.XmlSyntaxException&gt;.</source>
          <target state="translated">下表显示&lt;xref:System.Security.XmlSyntaxException&gt;.&lt;/xref:System.Security.XmlSyntaxException&gt;实例的初始属性的值</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">|属性 |值 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|内部异常引用。 |  |&lt;xref:System.Exception.Message%2A&gt;|错误消息字符串。 |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">解释异常原因的错误消息。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">导致当前异常的异常。</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`inner`</ph> parameter is not <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">如果<ph id="ph1">`inner`</ph>参数不是<bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>中, 引发当前异常<bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept>处理内部异常的块。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>